plugins/dynamodb-positions-storage/build.gradle

```diff
@@ -24,12 +24,15 @@ dependencies {
     compileOnly project(":api")
     compileOnly 'io.projectreactor:reactor-core'
 
-    compile ('com.amazonaws:aws-java-sdk-dynamodb:1.11.192') {
-        exclude group: "org.slf4j"
+    compile('software.amazon.awssdk:dynamodb:2.2.0') {
+        exclude group: 'org.slf4j'
     }
 
     testCompileOnly 'org.projectlombok:lombok'
     testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine'
     testCompile project(":tck")
     testCompile 'org.testcontainers:localstack'
+    testCompile('com.amazonaws:aws-java-sdk-dynamodb:1.11.475') {
```

Comment:
is it really needed now?

Reply:
localstack testcontainer uses v1 version for endpoint shortcut. In case I want to use testcontainers' container and those methods it have to be there :(

Reply:
I see. Hmmm, ok, let's keep it. Although I think there is no need to exclude slf4j from it (you have do to it for plugin's dependencies because this is an exposed API from the root module and plugins must link but not include it)